### ISSUE TITLE
'latin-1' codec can't encode character '\u2026' in position 513: ordinal not in range(256)

### ISSUE THREAD

**Issue by ayvacs** [clearclown/twitter_likes#3]
I am receiving a HTTP 500 error for all http requests, is there any way I can fix it?

This is my first time using Docker so sorry if it is a common error.

It happened with all usernames that I tested with.

<img width="641" alt="image" src="https://github.com/clearclown/twitter_likes/assets/74518502/2bc6b2ae-72f4-4f97-8724-ca31051ca72c">
<img width="552" alt="image" src="https://github.com/clearclown/twitter_likes/assets/74518502/109f6879-a60d-47de-9ec3-1a4a75d8f54d">

**Comment by babonihub** [comment]
Also have this issue

**Comment by miaou16** [comment]
I'm getting this error when i press "get likes" 
![image](https://github.com/clearclown/twitter_likes/assets/172699336/7504da32-9fdf-4bbf-a785-b04c290b5c49)

idk what im doing wrong :(

**Comment by miaou16** [comment]
Same. heard someone say they disabled the API but I hope they're wrong T_T

**Comment by alanneves** [comment]
yeah I think they removed the api, I'm getting this error when accessing the twitter endpoint directly
![image](https://github.com/clearclown/twitter_likes/assets/38226523/c51a234f-a9c6-44f6-a099-e8aca47c583c)

**Comment by swablueme** [comment]
I can't even see the Bearer token when I am looking at my own likes fug and twitter is detecting if the browser has js enabled or not. :(

**Comment by omniscentreader** [comment]
I'm having this same problem too :(

**Comment by Kevin-cwx** [comment]
Can someone confirm if its working for them. I got this error 
"Sorry, that page does not exist","code":34}]  - On a public page.

**Comment by IsLSteven** [comment]
the poster still hasn't appeared after 1 week? that's weird

**Comment by IsLSteven** [comment]
Guys, look, the .env doesn't have its data there? Would we have to replace it with ours? but how would we do that? I tried creating a fork and making mine .env but it still pulls his

**Comment by IsLSteven** [comment]
Here it also appears that the page was not found
![image](https://github.com/clearclown/twitter_likes/assets/123420330/00d00411-9634-4d79-9533-ad295adbb4ed)

**Comment by lorenzboni** [comment]
Any update on this please? I have just tried it (August 2024) and it just shows error: 500.
